app/main.py

```diff
@@ -2,0 +1,26 @@
+import os
+
+
+def move_file(command: str) -> None:
+    command = command.split(" ")
+    mv = command[0]
+    file_to_move = command[1]
+    destination = command[2].split("/")
+    if mv == "mv" and file_to_move != "".join(destination):
+        try:
+            with open(file_to_move, "r") as f:
+                data = f.read()
+            os.remove(file_to_move)
+        except FileNotFoundError:
+            print("File not found")
+        path = ""
+        if len(destination) == 1:
+            path = destination[0]
+        else:
+            for i in range(len(destination) - 1):
+                path += destination[i] + "/"
+                if not os.path.exists(path):
+                    os.mkdir(path)
+            path += destination[-1]
```

Comment:
Code Efficiency issue: You are not checking if the directory already exists before creating it. If you try to create a directory that already exists, it will lead to an error. You should check if the directory exists before trying to create it.